i18n/qqq.json

```diff
@@ -353,5 +353,14 @@
 	"srf-spreadsheet-link": "{{doc-smw-link}}",
 	"srf-paramdesc-spreadsheet-filename": "{{doc-paramdesc|filename}}",
 	"srf-paramdesc-spreadsheet-templatefile": "{{doc-paramdesc|templatefile}}",
-	"srf-paramdesc-icalendar-timezone": "{{doc-paramdesc|timezone}}"
+	"srf-paramdesc-icalendar-timezone": "{{doc-paramdesc|timezone}}",
+
+	"srf-paramdesc-diagramtitle": "{{doc-paramdesc|diagram}}",
+	"srf-paramdesc-diagramtheme": "{{doc-paramdesc|theme}}",
+	"srf-paramdesc-axisformat": "{{doc-paramdesc|axisformat}}",
+	"srf-paramdesc-sortkey": "{{doc-paramdesc|sortkey}}",
+	"srf-paramdesc-titletopmargin": "{{doc-paramdesc|titletopmargin}}",
+	"srf-paramdesc-barheight": "{{doc-paramdesc|barheight}}",
+	"srf-paramdesc-leftpadding": "{{doc-paramdesc|leftpadding}}",
+	"srf-paramdesc-bargap": "{{doc-paramdesc|bargap}}"
```

Comment:
Some docus are missing, e.g.
```
"srf-error-mapping-assignment": "This is an error message.\n\nParameter:\n* $1 holds ..."

"srf-error-mapping-keywords": "This is an error message."
"srf-printername-mermaid": "{{doc-smwformat|Gantt}}"
```
Sometimes the docu can be rather simple but still providing some context.

Reply:
Added missing docus on Commit:a7c6754d9d24145679a0336343024b86d2da82ca